### PR TITLE
Remove name from examples and update CLI version

### PR DESCRIPTION
**Pull request by jpe316**
# PR into Azure/azureml-v2-preview

## Checklist

I have:

- [x] read and followed the contributing guidelines

## Changes

-

Fixes #

### CODE REVIEW

File: examples/train/basic-command-job/command_job.yml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-name: command-job-example
-code:
-  directory: .
 command: /bin/sh -c 'pip freeze && echo hello world'
```
Comment (yanrez) [inline examples/train/basic-command-job/command_job.yml:1]: language server?

File: examples/train/basic-command-job/hello_python_job.yml [hunk 2]
```diff
@@ -1,4 +1,3 @@
-name: hello-python-example
 command: python hello.py
```
Comment (yanrez) [inline examples/train/basic-command-job/hello_python_job.yml:1]: language server here as well

File: examples/train/tensorflow/mnist/tf_mnist_job.yml [hunk 3]
```diff
@@ -1,8 +1,8 @@
-name: tf-mnist-example
+# yaml-language-server: $schema=https://azuremlsdk2.blob.core.windows.net/latest/commandJob.schema.json
 code: 
   directory: ./src
 command: python train.py
 environment: azureml:tf-env:1
 experiment_name: tf-mnist
```
Comment (yanrez) [inline examples/train/tensorflow/mnist/tf_mnist_job.yml:6]: is experiment_name here intentionally?

File: examples/train/tensorflow/mnist/tf_mnist_job.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
...
 compute:
-  target: azureml:testCompute
\ No newline at end of file
+  target: azureml:testCompute
```
Comment (yanrez) [inline examples/train/tensorflow/mnist/tf_mnist_job.yml:8]: goazurego?

File: examples/train/tensorflow/mnist-horovod/tf_horovod_job.yml [hunk 5]
```diff
@@ -9,4 +9,4 @@ distribution:
   process_count_per_instance: 2
 compute:
   target: azureml:testCompute
```
Comment (yanrez) [inline examples/train/tensorflow/mnist-horovod/tf_horovod_job.yml:11]: goazurego?

File: examples/train/tensorflow/mnist-distributed/tf_distr_job.yml [hunk 6]
```diff
@@ -9,4 +9,4 @@ distribution:
   worker_count: 2
 compute:
   target: azureml:testCompute
```
Comment (yanrez) [inline examples/train/tensorflow/mnist-distributed/tf_distr_job.yml:11]: goazurego

File: examples/train/fastai/pets-resnet34/fastai_pets_job.yml [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
+# yaml-language-server: $schema=https://azuremlsdk2.blob.core.windows.net/latest/commandJob.schema.json
 code: 
   directory: ./src
 command: python train.py
-environment: "azureml:fastai-vision:1"
+environment: azureml:fastai-vision:1
 compute:
-  target: "azureml:testCompute"
\ No newline at end of file
+  target: azureml:testCompute
```
Comment (yanrez) [inline examples/train/fastai/pets-resnet34/fastai_pets_job.yml:7]: goazurego

File: examples/train/tensorflow/mnist/tf_mnist_job.yml (hunk 3 above)
Comment (jpe316) [reply examples/train/tensorflow/mnist/tf_mnist_job.yml:6]: @mx-iao ?

**Review by yanrez — APPROVED** [review]

**Review by rihorn2 — APPROVED** [review]
looks good

**Pull request merged**